hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileStatus.java

```diff
@@ -201,6 +201,15 @@ public FsPermission getPermission() {
     return permission;
   }
 
+  /**
```

Comment:
comes in as part of HADOOP-14223. Is it needed? As this is such a core class I'm nervous about doing changes here except consistently with branch-3

Reply:
This was because I involved changes in hadoop-common between trunk and branch-2 due the reason above. I have redone it and it should be excluded now.